main.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "report_builder"
```

Comment:
```suggestion
require 'report_builder'
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>